lib/convert.js

```diff
@@ -0,0 +1,42 @@
+import fixGoogleHtml from './fix-google-html';
+// rehype-dom-parse is a lightweight version of rehype-parse that leverages
+// browser APIs -- reduces bundle size by ~200 kB!
+import parse from 'rehype-dom-parse';
+import { all } from 'rehype-remark';
+import rehype2remarkWithSpaces from './rehype-to-remark-with-spaces';
+import stringify from 'remark-stringify';
+import { unified } from 'unified';
+
+function preserveTagAndConvertContents (h, node) {
+  return [
+    h(node, 'html', `<${node.tagName}>`),
+    ...all(h, node),
+    h(node, 'html', `</${node.tagName}>`)
+  ];
+}
+
+const processor = unified()
+  .use(parse)
+  .use(fixGoogleHtml)
+  // .use(require('./lib/log-tree').default)
+  .use(rehype2remarkWithSpaces, {
+    handlers: {
+      // Preserve sup/sub markup; most Markdowns have no markup for it.
+      sub: preserveTagAndConvertContents,
+      sup: preserveTagAndConvertContents
+    }
+  })
+  .use(stringify, {listItemIndent: '1'});
+
+export function convertToMarkdown(html) {
```

Comment:
Given that part of the real goal here is to a) make this friendly for library-style usage and b) the naming issues around whether you are dealing with HTML or GDocs’s native representation (https://github.com/Mr0grog/google-docs-to-markdown/pull/40#issuecomment-1381086960), would you mind renaming this function to `convertDocsHtmlToMarkdown(html)` or something along those lines? I’d like to make sure this name:
1. Makes it clear this is expecting an HTML string, not a structured document.
2. Suggests something about the fact that this is Google-Docs-centric HTML, since common usage these days is `import { theFunction } from 'package'`, which leaves the function name divorced from the context of the package name (which is the part that currently tells you this is about Google Docs).

Reply:
renamed!